release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1051,61 +1355,371 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1907381[*BZ#1907381*])
+
+* When a default catalog source in the `openshift-marketplace` namespace is disabled through the OperatorHub API, you can a create custom catalog source with the same name as that default. Previously, custom catalog sources with the same name as a default catalog source were deleted by the Marketplace Operator when the marketplace was restarted. An annotation has been added to the default catalog sources that are created by the Marketplace Operator. Now the Operator only deletes the catalog sources that contain the annotation when the marketplace is restarted. Custom catalog sources created with the same name as the default catalog sources are not deleted. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1908431[*BZ#1908431*])
+
+* Previously, the `oc adm catalog mirror` command did not generate the proper mappings for Operator index images without namespaces. Additionally, the `--filter-by-os` option filtered the entire manifest list. This resulted in invalid references to the filtered images in the catalog. Index images without namespaces are now mapped correctly and an `--index-filter-by-os` option is added to filter only the index image that is pulled and unpacked. The `oc adm catalog mirror` command now generates valid mappings for index images without namespaces and the `--index-filter-by-os` option creates valid references to the filtered images. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1908565[*BZ#1908565*])
+
+* Previously, Operators could specify a `skipRange` in the cluster service version (CSV) replacement chain that would cause Operator Lifecycle Manager (OLM) to attempt to update the Operator with itself. This infinite loop would cause an increase in CPU usage. The CSV replacement chain is now updated so that Operators do not become stuck in an infinite loop due to an invalid `skipRange`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916021[*BZ#1916021*])
+
+* Previously, the `csv.status.LastUpdateTime` time comparison in the cluster service version (CSV) reconciliation loop always returned a `false` result. This caused Operator Lifecycle Manager (OLM) Operator to continuously update the CSV object and trigger another reconciliation event. The time comparison is now improved and the CSV is no longer updated when there are no status changes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917537[*BZ#1917537*]
```

Comment:
This is missing the ending parentheses after the BZ link.

Reply:
Also I think you might need a "the" in front of "Operator Lifecycle Manager (OLM) Operator" (or plural Operators if there are multiple)